index.html

```diff
@@ -9,8 +9,13 @@
 </head>
 
 <body>
-  <div id="app"></div>
+  <div id="app">
+    <div class="news">
```

Comment:
div is without meaning and should be use for grouping or if there is no other useful tag, maybe main or section? or something else ?